### DISCUSSION TITLE
I can no longer see my profile page

### DISCUSSION THREAD

**Discussion by instructorkjrsten** [beakerbrowser/beaker#1693] [General] [answered]
When I go "home" in Beaker Browser, I see a 'loading' icon, but no content.

Same when I try to view my Profile (or others' profiles). I can see userlist.beakerbrowser.com, but adding users to my Address Book is not working.

I'm on a Mac, my internet connection is good, and I could see and edit my files yesterday. What could be wrong?

(I am @kjtten in the directory)

**Comment by pfrazee** [comment, marked as answer]
Hey @instructorkjrsten, sorry you're having trouble here.

This sounds like an issue with the "hyperdrive daemon" which is the p2p system. (We're still fixing some cases where it bugs out.)

What I'd suggest is, close beaker and make sure it's not running in the task bar. If you've got a task manager you're comfortable with, check it to make sure no process is running named "beaker" or "hyperdrive." Then start beaker again and see if that solves it. LMK if not.

Also, can you share what version of Beaker you're running?

**Reply by instructorkjrsten** [reply]
Fixed! I did what you suggested, quit from the taskbar and then checked in Activity Monitor to make sure it wasn't running. When I reopened Beaker I saw it go through its initialization and I'm back!

I'm running the 1.0 Beta, downloaded on Thursday the 28th.

Thanks for your help! Looks like a really interesting project.
